### DISCUSSION TITLE
Login into Supabase with kakaotalk

### DISCUSSION THREAD

**Discussion by artifacts3** [supabase/supabase#11793] [Feature Requests]
# Feature request

## Is your feature request related to a problem? Please describe.

At the moment you can only login to Supabase with Google. But some korean people might want to use Kakao talk.

## Describe the solution you'd like

When you login, there should be an option to login in with KakaoTalk.

## Describe alternatives you've considered

Or any other providers that people might want that are stopping people from joining the Supabase community.

**Comment by kiwicopple** [comment]
Thanks @artifacts3 ! I'll convert this one to a Discussion for now and we can track the populariy (with upvotes). If you want to contribute this yourself (in the GoTrue repo) then we'd be happy to merge this in

**Comment by hwr12** [comment]
Hi. 
Since Kakao talk is a major login provider that is used by almost all Korean, adding kakao talk as login provider might be helpful reaching Korean devs attention to use supbase for their Projects.
